packages/@magic-sdk/provider/src/core/sdk-environment.ts

```diff
@@ -31,5 +31,5 @@ export function createSDK<SDK extends SDKBase>(
 
 export const sdkNameToEnvName = {
   'magic-sdk': 'magic-sdk' as const,
-  '@magic-sdk/react-native': 'magic-sdk-rn' as const,
+  '@magic-sdk/react-native-bare': 'magic-sdk-rn' as const,
```

Comment:
Could you make this mapping relationship to 
 '@magic-sdk/react-native': 'magic-sdk-rn-bare' as const,
So that it will be recorded in the Snowflake

Reply:
@Ethella That's the plan Stan! I purposely omitted this change so that we can test the changes. otherwise, this PR would have to utilize changes in `phantom` to get reviewed. 

I'll update it though.